app/components/candidate_interface/gcse_grade_guidance_component.html.erb

```diff
@@ -1,28 +1,15 @@
-<% unless @subject == 'english' %>
   <p class="govuk-body">
-    <% if gce_o_level? %>
-      <%= simple_format t('gcse_edit_grade.guidance.o_level_guidance'), class: 'govuk-body' %>
-    <% else %>
-      <%= simple_format t('gcse_edit_grade.guidance.main'), class: 'govuk-body' %>
+  <% if gce_o_level? %>
+    <%= simple_format t('gcse_edit_grade.guidance.o_level_guidance'), class: 'govuk-body' %>
+  <% elsif scottish_national_5? %>
+    <%= simple_format t('gcse_edit_grade.guidance.scottish_national_5'), class: 'govuk-body' %>
+  <% else %>
+    <%= simple_format t('gcse_edit_grade.guidance.main'), class: 'govuk-body' %>
+    <% if english? %>
+      <%= tag.p(t('gcse_edit_grade.guidance.multiple_english_gcses.secondary'), class: 'govuk-body') %>
```

Comment:
To keep things consistent it's best to stick to using the html tag format as it makes it easier to figure out what the output is

```suggestion
     <p class="govuk-body">
        <%= t('gcse_edit_grade.guidance.multiple_english_gcses.secondary') %>
     </p>
```